index.bs

```diff
@@ -6362,6 +6364,7 @@ in [[#es-buffer-source-types]].
 <pre class="grammar" id="prod-BufferRelatedType">
     BufferRelatedType :
         "ArrayBuffer"
+        "SharedArrayBuffer"
```

Comment:
This becomes a non-identifier token by virtue of appearing anywhere like this, so it needs to be added as an alternative for the `Other` production (used for the “anything goes” extended attribute cover grammar) as well.